### DISCUSSION TITLE
[OUTDATED] SparkVUE Webview

### DISCUSSION THREAD

**Discussion by Blobby-Boi** [3kh0/ext-remover#1012] [Webview JOIN TITANIUM NETWORK TO SHARE PLEASE]
**SPARKVUE IS NO LONGER ON THE WEBSTORE GO HERE FOR MORE INFORMATION: https://www.pasco.com/resources/blog/353**

So I made a version of my Blobbypass thing for SparkVUE using a patched exploit and sorta unpatching it.
This webview is actually pretty good since you don’t have to go through pages before using it, you can upload files, and cookies save if you restart your Chromebook!

Tutorial
1. Install [SparkVUE](https://chrome.google.com/webstore/detail/sparkvue/iimbdmgkimpbhimdjnmiffmeefbppijo) from the chrome webstore
2. Download [this file](https://tinyurl.com/bIobbypassSPK)
3. Open SparkVUE
4. Click “Open Saved Experiment”
5. Pick the file you just downloaded
6. Enter a URL and click Launch

Keep in mind that this doesn’t bypass network restrictions, so use caub/caudns/blobwifi or smth if you want to bypass that.

**Comment by xoidex** [comment]
hey is there any list of webview exploits or are they just kinda floating around

ty

**Reply by AshtonDavies** [reply]
Titanium Network has a large webview-kajigs channel.

You must follow the get-verified link to access kajigs if it is your first time.

**Reply by Blobby-Boi** [reply]
You can probably use the TN server or here’s a list of the apps that I remember:

SparkVUE (This lol)
Geogebra Classic
S2Bot4Scratch
McMillan learning ebook
Droneblocks 
myHomework student planner
3Dview
Coding with chrome
Mackinvia
Ipevo Whiteboard
TestNav

**Reply by Blobby-Boi** [reply]
There’s also other webviews that are not chrome webstore apps such as
Quickview (Patched in v115)
Save to google drive (Patched in v115)
Sync API (Patched in v115)
chrome://chrome-signin/
Epic Books (Google Play)
Lego WeDo 2.0 (Google Play)
Google Play Services (Requires Invalid session)
Google Play Settings

**Reply by xoidex** [reply]
ay thanks

i'm late

edit: just remembered droneblocks was allowed a few years ago
edit 2: just remembered i got the webstore version
edit 3: not sure why i'm still looking for a webview when i have a firewall bypass 💀

**Reply by Blobby-Boi** [reply]
lol

**Reply by Blobby-Boi** [reply]
wdym firewall bypass is that like ltmeat or smth

**Reply by xoidex** [reply]
no, a friend of a friend found a exploit that bypassed the school’s firewall on accident

**Reply by Blobby-Boi** [reply]
Cool

**Reply by gitthubb10** [reply]
Pretty sure no one will reply but xiodex, how do you do that (even though it prob won't work for me lol)

**Reply by Pleasehelpunblock** [reply]
Please give me some more apps or something my teacher blocked everything even apps and github and I don’t know how to send the file to my Chromebook please help all I have is sparkvue and I can’t even use it

**Reply by Blobby-Boi** [reply]
If you have sparkvue, then you should be able to use this webview regardless of whether github is blocked or not.

**Reply by jeffrey6576i** [reply]
well sparkvue is removed from chromeweb store because it violated googles policy (I swear its not blocked its completely removed)  
![Screenshot 2024-11-14 9 43 42 PM](https://github.com/user-attachments/assets/f47040a5-fae1-4a92-ac13-f8ae53202139)

**Comment by coolquiztestgenerator** [comment]
I'm pretty sure the link is old, because it is now [chromewebstore](https://chromewebstore.google.com)

**Reply by Blobby-Boi** [reply]
In the new webstore you can’t download chrome apps

**Comment by 3kh0** [comment]
Cool

**Reply by Pleasehelpunblock** [reply]
Please give me some more apps or something my teacher blocked everything even apps and github and I don’t know how to send the file to my Chromebook please help all I have is sparkvue and I can’t even use it

**Comment by Pleasehelpunblock** [comment]
Please give me some more apps or something my teacher blocked everything even apps and github and I don’t know how to send the file to my Chromebook please help all I have is sparkvue and I can’t even use it

**Reply by Blobby-Boi** [reply]
wdym you cant use it just open the link on ur chromebook

**Reply by Pleasehelpunblock** [reply]
Bro GitHub is blocked

**Reply by Pleasehelpunblock** [reply]
Idk what to do now

**Comment by Pleasehelpunblock** [comment]
I can’t use GitHub it is blocked

**Reply by Blobby-Boi** [reply]
That means that right now you are using another device right?

**Reply by Blobby-Boi** [reply]
If you are using another device then you can just copy the link to the [file](https://tinyurl.com/BIobbypassSPK) and then put that into ur chromebook since link isn't a github link

**Reply by Pleasehelpunblock** [reply]
Yes I have thought of using that idea and sent the file link to my Chromebook but when look at the email it’s just a link and when I try to paste it into browser it pops up as a blank page

**Reply by Blobby-Boi** [reply]
you don't have to send the link to your chromebook you can just type it in its not even that long

**Reply by Pleasehelpunblock** [reply]
I tried to type it into my Chromebook and it says I do not have access to the file error 403

**Reply by Pleasehelpunblock** [reply]
My school must be to good because I typed it in like that and it still says error 403 forbidden

**Reply by Pleasehelpunblock** [reply]
Nvm I finally got it thank you so much bro

**Reply by Blobby-Boi** [reply]
np

**Reply by jeffrey6576i** [reply]
@Blobby-Boi since sparkvue is gone... what now? sparkvue was one of the best apps you can literally use geforce on it

**Comment by synapt0** [comment]
does this still work?

**Reply by FunnyLoaf** [reply]
last I checked it did.

**Comment by Im1OfThem** [comment]
Bruh my school blocked it, all because it was secretly a website, do you guys have any backup files, cuz I want blobbypass back, or put the website under another name, or don't add any secret websites please bruh.

**Comment by VFXKLIPS** [comment]
Mine is blocked too

**Comment by Charlieis1nsan3** [comment]
hey Bobby can u tell me how this works I want to make my (with credit) I'm just confused how u did it?

**Reply by Charlieis1nsan3** [reply]
NVM I found out how you did it >:)

**Reply by Charlieis1nsan3** [reply]
BLOBBY >:)))

**Reply by awsome-webdev** [reply]
i found his exploit on *nettleweb*, a unblocked website, but was like tf is this black magic and then found the blobby.dev or something like that and some research later boom still only kind of have it. BUT i can clone it:

* [decoy login(desktop-notes.vercel.app)](https://desktop-notes.vercel.app)
* pres ctr+alt+l
* nothing will show up, it is just recording your keyboard in the background
* type: someexploit083572
*  hit enter
* then go to [https://desktop-notes.vercel.app/prq.html](https://desktop-notes.vercel.app/prq.html)
> you have to "*login*" every time you try to go to desktop-notes.vercel.app/prq.html or you will get redirected 
> thought this would help with not getting caught if you got pulled into principles office

**Reply by Charlieis1nsan3** [reply]
Lol 🤣 I saw that on the data url version. pretty cool

**Reply by awsome-webdev** [reply]
thnx, i was also wondering if their was a way to do script injection using the same payment request embeding technique

**Reply by Charlieis1nsan3** [reply]
Script injection can potentially occur if user inputs in payment request forms are not properly validated or sanitized, allowing scripts to be executed. Such as this Exploit right here.

**Reply by jeffrey6576i** [reply]
...

**Reply by awsome-webdev** [reply]
what exploit?😐

**Reply by jeffrey6576i** [reply]
sparkvue

**Reply by Charlieis1nsan3** [reply]
> what exploit?😐

Bruh BlobbypassXSS

**Reply by awsome-webdev** [reply]
oh

**Reply by VFXKLIPS** [reply]
> i found his exploit on _nettleweb_, a unblocked website, but was like tf is this black magic and then found the blobby.dev or something like that and some research later boom still only kind of have it. BUT i can clone it:
> 
> * [decoy login(desktop-notes.vercel.app)](https://desktop-notes.vercel.app)
> * pres ctr+alt+l
> * nothing will show up, it is just recording your keyboard in the background
> * type: someexploit083572
> * hit enter
> * then go to https://desktop-notes.vercel.app/prq.html
> 
> > you have to "_login_" every time you try to go to desktop-notes.vercel.app/prq.html or you will get redirected
> > thought this would help with not getting caught if you got pulled into principles office

How do you make it full screen

**Reply by S-PScripts** [reply]
> > i found his exploit on _nettleweb_, a unblocked website, but was like tf is this black magic and then found the blobby.dev or something like that and some research later boom still only kind of have it. BUT i can clone it:
> > 
> > * [decoy login(desktop-notes.vercel.app)](https://desktop-notes.vercel.app)
> > * pres ctr+alt+l
> > * nothing will show up, it is just recording your keyboard in the background
> > * type: someexploit083572
> > * hit enter
> > * then go to https://desktop-notes.vercel.app/prq.html
> > 
> > > you have to "_login_" every time you try to go to desktop-notes.vercel.app/prq.html or you will get redirected
> > > thought this would help with not getting caught if you got pulled into principles office
> 
> How do you make it full screen

join our Discord server! (Pinned post)

**Reply by jeffrey6576i** [reply]
> > > i found his exploit on _nettleweb_, a unblocked website, but was like tf is this black magic and then found the blobby.dev or something like that and some research later boom still only kind of have it. BUT i can clone it:
> > > 
> > > * [decoy login(desktop-notes.vercel.app)](https://desktop-notes.vercel.app)
> > > * pres ctr+alt+l
> > > * nothing will show up, it is just recording your keyboard in the background
> > > * type: someexploit083572
> > > * hit enter
> > > * then go to https://desktop-notes.vercel.app/prq.html
> > > 
> > > > you have to "_login_" every time you try to go to desktop-notes.vercel.app/prq.html or you will get redirected
> > > > thought this would help with not getting caught if you got pulled into principles office
> > 
> > 
> > How do you make it full screen
> 
> join our Discord server! (Pinned post)
where is the pinned post

**Reply by S-PScripts** [reply]
> > > > i found his exploit on _nettleweb_, a unblocked website, but was like tf is this black magic and then found the blobby.dev or something like that and some research later boom still only kind of have it. BUT i can clone it:
> > > > 
> > > > * [decoy login(desktop-notes.vercel.app)](https://desktop-notes.vercel.app)
> > > > * pres ctr+alt+l
> > > > * nothing will show up, it is just recording your keyboard in the background
> > > > * type: someexploit083572
> > > > * hit enter
> > > > * then go to https://desktop-notes.vercel.app/prq.html
> > > > 
> > > > > you have to "_login_" every time you try to go to desktop-notes.vercel.app/prq.html or you will get redirected
> > > > > thought this would help with not getting caught if you got pulled into principles office
> > > 
> > > 
> > > How do you make it full screen
> > 
> > 
> > join our Discord server! (Pinned post)
> > where is the pinned post

repinned

**Comment by dudedecliedddddddd** [comment]
why don't it work now? I was using it last week and no I didnt update my chromebook or anything

**Reply by Charlieis1nsan3** [reply]
Your school is using DNS blocking forcing it not to work, it happens at my school also. So if you are able to just pull it up on a home network and then it should work at school

**Reply by jeffrey6576i** [reply]
just that?

**Reply by Charlieis1nsan3** [reply]
bruh not just that. people have had it previously downloaded retard.

**Reply by dudedecliedddddddd** [reply]
i use it at home not at school I tried the pwa app and it didn't work also and it still works on the sparkvue mobile app

**Reply by dudedecliedddddddd** [reply]
turns out you cant use it anymore if u deleted the app , my dumb ahh deleted it to reinstall because I thought the app was the issue 💀

**Comment by schoolexploitkid** [comment]
yes, sparkvue got removed. https://www.pasco.com/resources/blog/353 this is from the company that owns it

**Reply by Charlieis1nsan3** [reply]
bruh everybody knows that. but their are people who previously downloaded it before it was removed

**Reply by jeffrey6576i** [reply]
and you cant even use the web version too its ass

**Comment by dudedecliedddddddd** [comment]
bro before they removed the app some kid at my school were selling the method on how to do it💀 they even had to pay him to put on blobwifi

**Reply by jeffrey6576i** [reply]
thats fucking stupid

**Reply by S-PScripts** [reply]
> bro before they removed the app some kid at my school were selling the method on how to do it💀 they even had to pay him to put on blobwifi

that's fucking smart (he about be rich)

**Reply by awsome-webdev** [reply]
> > bro before they removed the app some kid at my school were selling the method on how to do it💀 they even had to pay him to put on blobwifi
> 
> that's fucking smart (he about be rich)

people at my school wanted to pay me for pirated offline music on a sd card because spotify got blocked

**Reply by S-PScripts** [reply]
ok

**Comment by Im1OfThem** [comment]
There is a shared file for Minecraft, so why not for fortnite? If someone has it, please send it to me, I'm struggling on proxies since my school somehow blocked everything, even IP banning every type of proxy known to man.

**Reply by S-PScripts** [reply]
Join Our Discord For More! https://github.com/3kh0/ext-remover/discussions/1532

**Reply by S-PScripts** [reply]
> youtubeunblocked.live

doesnt even work

**Reply by Blobby-Boi** [reply]
because the minecraft files are a port to javascript called eaglercraft and it would be much MUCH harder to port something like fortnite

**Reply by Charlieis1nsan3** [reply]
@Blobby-Boi and plus the Fortnite source code is very open-source

**Reply by Charlieis1nsan3** [reply]
isn't*

**Comment by jeffrey6576i** [comment]
wow @Blobby-Boi you made me watch a 3 minute video just to get in grr I want a refund

**Reply by S-PScripts** [reply]
wut?

**Reply by Blobby-Boi** [reply]
@S-PScripts I did an april fools where they had to watch the whole never gonna give you up video and type "I am gay" to use it 😆

**Reply by S-PScripts** [reply]
> @S-PScripts I did an april fools where they had to watch the whole never gonna give you up video and type "I am gay" to use it 😆

how does jeffrey not know what a rickroll is 😭

**Reply by jeffrey6576i** [reply]
> > @S-PScripts I did an april fools where they had to watch the whole never gonna give you up video and type "I am gay" to use it 😆
> 
> how does jeffrey not know what a rickroll is 😭

I DO KNOW WHAT IT IS

**Reply by S-PScripts** [reply]
> > > @S-PScripts I did an april fools where they had to watch the whole never gonna give you up video and type "I am gay" to use it 😆
> > 
> > 
> > how does jeffrey not know what a rickroll is 😭
> 
> I DO KNOW WHAT IT IS

why did you say '3 minute video' instead of 'rickroll' then ???

**Reply by jeffrey6576i** [reply]
> > > > @S-PScripts I did an april fools where they had to watch the whole never gonna give you up video and type "I am gay" to use it 😆
> > > 
> > > 
> > > how does jeffrey not know what a rickroll is 😭
> > 
> > 
> > I DO KNOW WHAT IT IS
> 
> why did you say '3 minute video' instead of 'rickroll' then ???

was I supposed to?

**Reply by S-PScripts** [reply]
> > > > > @S-PScripts I did an april fools where they had to watch the whole never gonna give you up video and type "I am gay" to use it 😆
> > > > 
> > > > 
> > > > how does jeffrey not know what a rickroll is 😭
> > > 
> > > 
> > > I DO KNOW WHAT IT IS
> > 
> > 
> > why did you say '3 minute video' instead of 'rickroll' then ???
> 
> was I supposed to?

it's just weird because it's not like a rickroll is random, it's a popular meme.

**Reply by jeffrey6576i** [reply]
that wasn't my question

> > > > > > @S-PScripts I did an april fools where they had to watch the whole never gonna give you up video and type "I am gay" to use it 😆
> > > > > 
> > > > > 
> > > > > how does jeffrey not know what a rickroll is 😭
> > > > 
> > > > 
> > > > I DO KNOW WHAT IT IS
> > > 
> > > 
> > > why did you say '3 minute video' instead of 'rickroll' then ???
> > 
> > 
> > was I supposed to?
> 
> it's just weird because it's not like a rickroll is random, it's a popular meme.

**Reply by S-PScripts** [reply]
> that wasn't my question
> 
> > > > > > > @S-PScripts I did an april fools where they had to watch the whole never gonna give you up video and type "I am gay" to use it 😆
> > > > > > 
> > > > > > 
> > > > > > how does jeffrey not know what a rickroll is 😭
> > > > > 
> > > > > 
> > > > > I DO KNOW WHAT IT IS
> > > > 
> > > > 
> > > > why did you say '3 minute video' instead of 'rickroll' then ???
> > > 
> > > 
> > > was I supposed to?
> > 
> > 
> > it's just weird because it's not like a rickroll is random, it's a popular meme.

You weren't supposed to/you didn't need to, but it's odd that you referenced to it as a random video instead of a rickroll since the rickroll video has 1b+ views...

**Reply by jeffrey6576i** [reply]
> > that wasn't my question
> > > > > > > > @S-PScripts I did an april fools where they had to watch the whole never gonna give you up video and type "I am gay" to use it 😆
> > > > > > > 
> > > > > > > 
> > > > > > > how does jeffrey not know what a rickroll is 😭
> > > > > > 
> > > > > > 
> > > > > > I DO KNOW WHAT IT IS
> > > > > 
> > > > > 
> > > > > why did you say '3 minute video' instead of 'rickroll' then ???
> > > > 
> > > > 
> > > > was I supposed to?
> > > 
> > > 
> > > it's just weird because it's not like a rickroll is random, it's a popular meme.
> 
> You weren't supposed to/you didn't need to, but it's odd that you referenced to it as a random video instead of a rickroll since the rickroll video has 1b+ views...

bro I know what it is I been rickrolled before..
